cmd/cmd.go

```diff
@@ -914,8 +908,25 @@ func checkServerHeartbeat(_ *cobra.Command, _ []string) error {
 	return nil
 }
 
+func versionHandler(_ *cobra.Command, _ []string) {
+	fmt.Printf("ollama version %s\n", version.Version)
+
+	client, err := api.ClientFromEnvironment()
+	if err != nil {
+		return
+	}
+
+	serverVersion, err := client.Version(context.TODO())
```

Comment:
You could use the cmd.Context here too by updating the arguments

Reply:
Good catch. I did this making the other context changes so must have missed it